### ISSUE TITLE
I broked the theme on my system

### ISSUE THREAD

**Issue by CarterSnich** [grassmunk/Chicago95#330]
Do you guys know what happened here? I was supposed to copy `./themes/Chicago95` to `/usr/themes/`, but unknowingly used `mv` command instead of `cp`.

![2024-02-06_09-09](https://github.com/grassmunk/Chicago95/assets/52433531/7bed8912-a318-4e64-b4db-baa7725d0ca4)

**Comment by grassmunk** [comment]
Do you log out and back in and reset the theme?

**Comment by CarterSnich** [comment]
Yes. But, I decided to just use Arc Dark. How can I reset the fonts? Some websites still show Helvetica font on Firefox. 

![image](https://github.com/grassmunk/Chicago95/assets/52433531/976d4938-0f04-4ea8-93c8-3b60fdc9c0b5)

But I want to keep this setting on.
![image](https://github.com/grassmunk/Chicago95/assets/52433531/059d5bd6-81c5-4e87-a6c4-c4b3f103445c)

**Comment by erned98** [comment]
@CarterSnich - I might be a little late with the answer but I fixed the same issue just now.

If you had to unblock bitmap font support on your EnOS to use Helvetica, you need to revert this decision now. This command helped for me:
`sudo mv /etc/fonts/conf.d/70-no-bitmaps.conf.bak /etc/fonts/conf.d/70-no-bitmaps.conf`

**Comment by Sid009hey** [comment]
did u fix the issue if so how. im facing this same issue.

**Comment by Sid009hey** [comment]
fixed mine, insta was referencing the web standard Helvetica, but since the system had a different weirdly rendered helvetica. it took that. deleting helvetica and the added fonts fixed the issue.
